website/frontend/templates/docs/build-windows.html

```diff
@@ -118,7 +118,9 @@ <h3 id="python2_packages">Python 2 packages</h3>
       </p>
 
       <ul>
-        <li>PyQt4 - Download and run the installer from <a rel="nofollow" target="_blank" href="https://sourceforge.net/projects/pyqt/files/PyQt4/PyQt-4.11.4/">https://sourceforge.net/projects/pyqt/files/PyQt4/PyQt-4.11.4/</a>.</li>
+        <li>PyQt4 - Riverside Computing (authors of PyQt) no longer maintain an installation executable.
```

Comment:
This doesn't need an update because River*bank* Computing didn't provide an installer on their website when these instructions were last updated (http://web.archive.org/web/20170404123507/https://riverbankcomputing.com/software/pyqt/download).

Reply:
I agree - this is what I worked out so not sure why I made this change. Will revert.

Reply:
Looked at this again, and what it does is keep the link to sourceforge but provide a more detailed explanation. So I think it should be left in - though I am about to tweak the explanation.